src/resources/css/IntroScreen.less

```diff
@@ -0,0 +1,62 @@
+.IntroScreen{
+
+/*variables*/
+@gmGreen: #BCFFBA;
+
+/*mixins*/
+.centreChildren{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+/*Page header*/
+header{
+    position: absolute;
+    width: 100%;
+}
+
+header a{
+    font-size: 20px;
+    margin: 10px;
+    cursor: pointer;
+}
+
+#imgAvatar{
+    height: 20px;
+    margin: 10px;
+}
+
+#containerLanding{
+    .centreChildren;
+    width: 100%;
+    height: 100%;
+}
+
+/*Center screen contents*/
+#imgTitleLogo{
+    width: 75px;
+    display: block;
+    margin: 1%;
+}
+
+#imgTitleText{
+    width: 800px;
+    display: block;
+}
+
+#containerLanding button{
+    margin: 10px;
+    width: 100px;
+    color: darkgreen;
+    font-size: 15px;
+    font-family:sans-serif;
+    font-weight: 400;
```

Comment:
Die fonts op die button hoort nie hier nie tensy hierdie die enigste button in die site gaan wees wat so moet lyk. So hoe jy 'n button wil he, maak hom so lyk in main.css en apply daai class.